include/rocksdb/db.h

```diff
@@ -127,6 +127,7 @@ struct IngestExternalFileArg {
   IngestExternalFileOptions options;
   std::vector<std::string> files_checksums;
   std::vector<std::string> files_checksum_func_names;
+  std::vector<Temperature> file_temperatures;
```

Comment:
One question is that whether we need temperature per file or per load? I think allowing single files to have different temperature in one single load might be over-engineering. This is also prone to misuse.

Also, adding more std::vector doesn't feel scalable to me. Sooner or later, we might want to refine the interface with something like `std::vector<FileInfo>` where FileInfo is a struct containing file name, checksums information and temperature. I think it might be worth adding to it.

Reply:
So your suggestion is, just give one Temperature variable in the `struct IngestExternalFileArg` for now, and add the comments. Later on, we can just refactor the struct with `std::vector<FileInfo>` to store all the file related informaiton?

Reply:
It sounds good.